packages/lesswrong/lib/collections/conversations/schema.ts

```diff
@@ -116,7 +115,54 @@ const schema: SchemaType<"Conversations"> = {
       );
       const filteredMessage = await accessFilterSingle(currentUser, context.Messages, message, context)
       return filteredMessage;
-    }
+    },
+    sqlResolver: ({field, join}) => join({
```

Comment:
This SQL resolver isn't strictly needed for this PR, but I figured I'd add it whilst I'm here anyway as it means we need one less Postgres query to load the inbox.